_authors/hdhalter.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Heather Halter
+short_name: hdhalter
+photo: '/assets/media/authors/hdhalter.jpg'
```

Comment:
looks like the file uploaded is .jpeg but this references .jpg

Reply:
so sorry. fixing now!